packages/blocks/google-cloud/src/lib/google-cloud-cli.ts

```diff
@@ -0,0 +1,48 @@
+import { runCliCommand } from '@openops/common';
+import fs from 'fs/promises';
+import os from 'os';
+import path from 'path';
+
+export async function runCommand(
+  command: string,
+  auth: any,
+  shouldUseHostCredentials: boolean,
+  project?: string,
+): Promise<string> {
+  const envVars: Record<string, string> = {
+    PATH: process.env['PATH'] || '',
+  };
+
+  const processGoogleCloudConfigDir = process.env['CLOUDSDK_CONFIG'];
```

Comment:
I'll need to make a follow up pr to actually add the env variable in all the places similarly to what @MarceloRGonc did with AZURE_CONFIG_DIR